test/lib/allocator_test.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+require_relative '../test_helper'
+
+describe GlobalUid::Allocator do
+  let(:connection)   { mock('connection') }
+  let(:increment_by) { 10 }
+  let(:allocator)    { GlobalUid::Allocator.new(incrementing_by: increment_by, connection: connection) }
+  let(:table)        { mock('table_class').class }
+
+  before do
+    restore_defaults!
+    connection.stubs(:current_database).returns('database_name')
+  end
+
+  describe 'when the auto_increment_increment differs to configuration' do
+    it 'prevents the connection to avoid allocating bad IDs' do
+      connection.stubs(:select_value).with('SELECT @@auto_increment_increment').returns(50)
+      exception = assert_raises(GlobalUid::InvalidIncrementException) do
```

Comment:
I think you can assert the message here, too:
```suggestion
      assert_raises(GlobalUid::InvalidIncrementException, "Configured: '10', Found: '50' on 'database_name'") do
```

Reply:
The line may be too long, but I think it’s worth it in this case.

Reply:
I thought that too, [but it doesn't](https://github.com/seattlerb/minitest/blob/6257210b7accfeb218b4388aaa36d3d45c5c41a5/lib/minitest/assertions.rb#L372-L392)